spec/acceptance/sensu_environment_spec.rb

```diff
@@ -1,72 +0,0 @@
-require 'spec_helper_acceptance'
```

Comment:
Since the filenames follow a pattern following the types, perhaps this should be a file that is just a comment that lists the issue number and lets you know the tests are in `spec/acceptance/sensu_rbac_spec.rb`. What do you think?

Reply:
I'm worried it will look like tests are missing and it will not be clear if these types are modified where the tests are.

Reply:
I think empty files should be fine, with a comment pointing back to sensu_rbac_spec.rb.